sklearn/tree/_tree.pyx

```diff
@@ -173,17 +265,19 @@ cdef class ClassificationCriterion(Criterion):
         self.weighted_n_left = 0.0
         self.weighted_n_right = 0.0
 
+        # Count labs for each output
         self.label_count_left = NULL
         self.label_count_right = NULL
         self.label_count_total = NULL
-
-        # Count labels for each output
         self.n_classes = NULL
+
         safe_realloc(&self.n_classes, n_outputs)
 
         cdef SIZE_t k = 0
         cdef SIZE_t label_count_stride = 0
 
+        # For each target, set the number of unique classes in that target,
+        # and also set the stride for that target
```

Comment:
`and also set the stride for that target`-> `and also compute the maximal stride of all target`